docs/outgoing-webhooks.md

```diff
@@ -0,0 +1,113 @@
+# Outgoing Webhooks
+
+Outgoing webhooks are a type of integration which post data to third party URLs through a HTTP POST request.
```

Comment:
Outgoing webhooks are a type of integration where the Zulip server sends HTTP POST requests to a third party URL.